pyms/flask/services/metrics.py

```diff
@@ -30,9 +30,14 @@ def before_request(self):  # pylint: disable=R0201
         request.start_time = time.time()
 
     def after_request(self, response):
+        if hasattr(request.url_rule, "rule"):
```

Comment:
@avara1986 Do you know if there's a real possibility that a request wouldn't have this attribute? I've tested locally and even without using dynamic values for the endpoints works. But the test doesn't, without the conditional. I'm not sure if it really should be like this and have a fallback or is the test_client that's not really complete.

Reply:
I figure out this weekend :)

Reply:
@alexppg the problem is not in tests. `request.url_rule` is None when you get 404 response. It makes sense if you visit a non-exists page therefore, you haven't a url_rule. 

PR approved

PD: sorry, one week late... :see_no_evil: